applications/openshift/high-availability/control_plane_nodes_in_three_zones/rule.yml

```diff
@@ -0,0 +1,54 @@
+documentation_complete: true
+
+title: 'Ensure control plane / master nodes are distribute across three failure zones'
+
+description: |- 
+    Distributing Kubernetes control plane nodes across failure zones enhances security by mitigating
+    the risk of a single point of failure and reducing the impact of API inavailability or attacks
+    targeting a specific zone. Maintaining three instances of etcd across different failure zones 
+    ensures fault tolerance and enables the system to reach quorum, thereby safeguarding critical data
+    integrity and ensuring continued operation even in the event of zone failures.
+
+rationale: |-
+    Distributing Kubernetes control plane nodes across failure zones is crucial for enhancing overall 
+    system resilience and security. By spreading control plane components across different zones, 
+    the system becomes more fault-tolerant, reducing the risk of widespread outages due to failures or 
+    attacks in a single zone. Having multiple instances of etcd spread across these zones ensures data 
+    integrity and availability, as it requires a quorum of nodes to reach consensus.
+    With three zones, Kubernetes can achieve a quorum with a simple majority (i.e., two out of three) 
+    for critical components like etcd, ensuring system stability even if one zone fails. 
+    Failure zones are marked on nodes using a well-known label called "topology.kubernetes.io/zone". 
+    This label is automatically assigned to each node by cloud providers but might need to be managed
+    manually in other environments
+
+identifiers: {}
```

Comment:
unique identifier is missing, add it by using 
`./utils/rule_dir_json.py` followed by `python utils/fix_rules.py --product products/ocp4/product.yml add-cce --cce-pool redhat control_plane_nodes_in_three_zones`